server/drivers/MD8800.c

```diff
@@ -3,7 +3,7 @@
  */
 
 /*
-    Copyright (C) 2006 Stefan Herdler in collaboration with Martin M�ller.
+    Copyright (C) 2006 Stefan Herdler in collaboration with Martin M�ller.
```

Comment:
Is this change on purpose? GitHub now renders the character wrongly (and did correctly before).

Reply:
No, it seems my editor messed up the file encoding (possible during a regex replace). I will fix all these issues.